test/integration/search/test_search.js

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Yay - a test! But right now it's not run automatically because it's an 'integration' test.

`make test` runs `unit, functional, localization` (no integration)
`make smoke` only runs `integration/smoke-testing`, so anything that is just in `integration` falls through the cracks. 

I also just noticed that there are no `functional` tests. Perhaps they were removed at some point.

Is this test something that should run locally for developers, or only as part of an automated smoke test?

Reply:
Thanks for pointing this out. I'm moving the test into `smoke-testing`, though it doesn't need the full smoke environment. 

I also filed this issue re: integration tests not running: https://github.com/LLK/scratch-www/issues/2992